client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -31,7 +35,7 @@ export const WarningList = ( { context, translate, warnings, showContact = true
 			{ warnings.map( ( { name, description, supportUrl, domainNames }, index ) => (
 				<div className="eligibility-warnings__warning" key={ index }>
 					<div className="eligibility-warnings__message">
-						{ context !== 'plugin-details' && (
+						{ context !== 'plugin-details' && context !== 'hosting-features' && (
```

Comment:
I'm hiding the message name returned from the backend, for hosting features, to match the design.